reference/auth/models/assertionverify.v1.yaml

```diff
@@ -0,0 +1,34 @@
+title: Assertion Verify
+description: Request body for verifying an assertion.
+type: object
+properties:
+  assertion:
+    type: string
+    pattern: '^(?:[A-Za-z0-9+/]{4})*(?:[A-Za-z0-9+/]{2}==|[A-Za-z0-9+/]{3}=)?$'
+    description: Base64 encoded data received from Apple App Attest API. User must base64 encode the binary data received from Apple.
+  clientData:
+    type: object
+    properties:
+      challenge:
+        description: The previously returned assertion challenge.
+        type: string
+        minLength: 1
+      partner:
+        description: Code name of partner to retrieve certificate from.
+        type: string
+        minLength: 1
+        enum:
+          - Coastal
+          - PalmTree
+      partnerData:
+        oneOf:
+          - $ref: './coastalrequest.v1.yaml'
+          - $ref: './palmtreerequest.v1.yaml'
+    description: Actual data requested by client. Must include the previously requested challenge.
+  keyId:
+    $ref: './keyid.v1.yaml'
+    description: Base64 encoded key Id received from Apple App Attest API.
```

Comment:
```suggestion
```
This property will be dropped, as you used a $ref as a sibling. So the description field in the `keyId.v1.yaml` file will be used instead.